src/transforms/postcss.js

```diff
@@ -48,6 +48,15 @@ async function getConfig(asset) {
     config.plugins.push(postcssModules(postcssModulesConfig));
   }
 
+  if (config.parser === undefined && asset.options.css_parser !== undefined) {
+      const name = asset.options.css_parser;
+
+      if (!(typeof name === "string")) throw "Internal Error: Invalid PostCSS parser type. Should be string";
+
+      let parser = await localRequire(name, asset.name).catch(() => require(name));
```

Comment:
why does it catch with a require, that's what localRequire already does?

localRequire's should just throw if it fails, because require is just the same thing. (Unless you want to require something from inside parcel, but that should happen inside the transform not be a fallback as this could have unwanted side-effects)